gef-ui/server/controllers/application-details/index.test.js

```diff
@@ -280,6 +282,18 @@ describe('controllers/application-detaills', () => {
             applicationStatus: mockApplicationResponse.status,
           }));
       });
+
+      it('renders `unissued-facilities` when page requested is `unissued facilities` ', async () => {
+        mockApplicationResponse.status = CONSTANTS.DEAL_STATUS.UKEF_ACKNOWLEDGED;
+        api.getApplication.mockResolvedValueOnce(mockApplicationResponse);
+
+        await applicationDetails(mocks.MockRequestUrl('/gef/application/123/unissued-facilities'), mockResponse);
+
+        expect(mockResponse.render)
+          .toHaveBeenCalledWith('partials/unissued-facilities.njk', expect.objectContaining({
+            applicationStatus: mockApplicationResponse.status,
+          }));
```

Comment:
Could we add the extra fields passed to template? 

`unissuedFacilitiesPresent`, `facilitiesChangedToIssued`

Reply:
done